examples/show_large_tree.py

```diff
@@ -6,13 +6,15 @@
 from copy import copy
 
 import napari
+import numpy as np
 
 from napari_arboretum.graph import TreeNode
 
 
 def generate_binary_tree(max_depth: int):
     track_id = 0
-    nodes = [TreeNode(track_id, t=(0, 1), generation=1)]
+    t = np.array([0, 1])
```

Comment:
Nice way of doing it!

Reply:
it was actually necessary because mypy was complaining about `t` being a tuple

Reply:
And rightly so